playgame.py

```diff
@@ -24,6 +25,17 @@ def main(argv):
     else:
         run_one_game(args)
 
+def create_dirs(path):
```

Comment:
please just use `makedirs`. there is no need to separate the directories yourself

Reply:
To be clear, I'm asking to clean up create_dirs, not remove it